fedora_commons/Dockerfile

```diff
@@ -1,14 +1,21 @@
-FROM tomcat:8.5
+FROM tomcat:9
+
+ENV FCREPO4_HOME=/fcrepo4_home
+ENV FCREPO4_DATA=/fcrepo4_data
+ENV CATALINA_BASE=$FCREPO4_HOME
+
+RUN mkdir -p $FCREPO4_HOME/bin $FCREPO4_HOME/conf $FCREPO4_HOME/lib $FCREPO4_HOME/logs $FCREPO4_HOME/webapps $FCREPO_HOME/work $FCREPO4_HOME/temp
+COPY conf $FCREPO4_HOME/conf
+COPY setenv.sh $CATALINA_HOME/bin/setenv.sh
+
+ADD https://github.com/fcrepo4/fcrepo4/releases/download/fcrepo-4.7.2/fcrepo-webapp-4.7.2.war $FCREPO4_HOME/webapps/ROOT.war
```

Comment:
You can allow for the large files to be downloaded locally to speed repeated executions of this step.

The Dockerfile logic would need to go a bit like:

if present locally (e.g. in a directory like /cache which would have been .gitignore-d). ADD the local copy to the image being built
if not present locally, download the file locally (to /cache). ADD the local copy to the image being built